google.json

```diff
@@ -798,7 +982,7 @@
     {
         "id": "com.google.android.dialer",
         "label": "Phone",
-        "description": "Formerly Google Dialer.\nDefault dialer on some phones.\nGoogle Analytics are embedded in the app, assume everything is datamined.",
+        "description": "Formerly Google Dialer.\nDefault dialer on some phones.\nGoogle Analytics are embedded in the app, assume everything is datamined.\nSimple Dialer is a good FOSS replacement based on the AOSP Dialer.",
```

Comment:
We don't suggest stuffs in the description. There's a dedicated section for it (which is already filled).